src/test/logic/operate.test.js

```diff
@@ -0,0 +1,33 @@
+import Operation from '../../logic/operate';
+
+describe('Operate', () => {
+  test('should sum two numbers', () => {
+    const answer = Operation('4', '5', '+');
+    expect(answer.toString()).toEqual('9');
```

Comment:
- [ ] Please divide this and any other test that has many expects inside into multiple tests with only 1 expect inside each one.